src/waveform/widgets/glslwaveformwidget.cpp

```diff
@@ -64,12 +64,8 @@ GLSLWaveformWidget::GLSLWaveformWidget(
     m_initSuccess = init();
 }
 
-GLSLWaveformWidget::~GLSLWaveformWidget() {
-    makeCurrent();
-}
-
 void GLSLWaveformWidget::castToQWidget() {
-    m_widget = static_cast<QWidget*>(static_cast<QGLWidget*>(this));
+    m_widget = this;
```

Comment:
Is this change necessary? ba340abe491f5dd7e9e8b4cd5296ab64e21d7a98#diff-022b8a0f41d44be564c4ef50a719a09cL56 did not do it for the other GL waveforms.

Reply:
It builds, so I guess it's okay.

Reply:
Needless redundancy. IDK why this was necessary.